dali/test/python/test_operator_python_function.py

```diff
@@ -319,3 +330,18 @@ def test_wrong_outputs_number():
         print(e)
         return
     raise Exception('Should not pass')
+
+
+counter = 0
+
+
+def increase(images):
+    global counter
```

Comment:
Weren't there some issues with synchronization and `global`? @awolant?

Reply:
There were. That's why the test pipeline is run only with one worker thread. I think we should not support this kind of usage of python operator for now, but this was the easiest way to test if the operator wasn't pruned.

Reply:
It should work like this with one thread, but for someone not familiar with the issue it's not immediately obvious why, I think. And with this precedence here someone will try to do something that we don't support.
Proposed solution: let the function create a file in '/tmp/sink_test' and let the assert be counting those files and comparing to `BATCH_SIZE`

Reply:
It should work like this with one thread, but for someone not familiar with the issue it's not immediately obvious why, I think. And with this precedence here someone will try to do something that we don't support.
Proposed solution: let the function create a file in '/tmp/sink_test' and let the assert be counting those files and comparing to `BATCH_SIZE`